README.md

```diff
@@ -95,6 +95,22 @@ During installation private certificates, public certificates and configuration
 # Adding nodes
 To add a node to an existing cluster is as easy as adding it to the [inventory](https://docs.ansible.com/ansible/latest/user_guide/intro_inventory.html) and running `install.yml` again.
 
+# High availability
```

Comment:
I think this header could be renamed to "Rolling Upgrades"

Reply:
Done!